commons/tool/src/webapp/WEB-INF/bootstrap.jsp

```diff
@@ -41,7 +41,9 @@
         <div id="Mrphs-sakai-commons" class="portletBody commons-portletBody">
 
             <ul id="commons-toolbar" class="navIntraTool actionToolBar hidden" role="menu"></ul>
-
+            <div class="page-header">
+                <h1>Commons</h1>
```

Comment:
This title needs to be internationalized, so it should be sourced from a message bundle.

Reply:
@fostersdesign  If the message bundle is for the Commons string, don't worry about it at the moment. Open another JIRA just about that and I'll pick it up. Unless William wants to look at that particularly - message bundles can be a bit odd, with class loaders and all.